dags/common/request.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+import requests
+
+
+class Request:
+    def __init__(
+        self,
+        base_url: str = "localhost",
+        port: int = 8080,
+        parameters: dict = {},
+        headers: dict = {},
+        path_segments: list = [],
+    ):
+        self.base_url = base_url
+        self.parameters = parameters
+        self.headers = headers
+        self.path_segments = path_segments
+
+    def __repr__(self) -> str:
+        return f"< RestApi base_url={self.base_url}, parameters={self.parameters}, header={self.headers}, path_segments={self.path_segments} >"
```

Comment:
same this should be wrapped in multiple lines

Reply:
it passes pre-commit :/

Reply:
probably because you edit this file before you install pre-commit, try to modify the file and commit, it should fix it